### DISCUSSION TITLE
URGENT: Enterprise account locked out of support for 5+ days

### DISCUSSION THREAD

**Discussion by kkodmani** [community/community#57624] [Enterprise] [answered]
Hello, 
I have been rehired with the same enterprise and I am unable to access my work GitHub account because I lost 2FA and password months ago. 
I have a pending ticket 2189712 that cannot receive enterprise support that has been open for 5 business days now.

The organization suggested the only way to solve this is to contact GitHub support.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @kkodmani, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
